.jenkins/Jenkinsfile-cryptography-wheel-builder

```diff
@@ -120,11 +161,25 @@ for (config in configs) {
     for (_version in versions) {
         def version = _version
 
-        def combinedName = "${label}-${version}"
-        builders[combinedName] = {
-            node(label) {
-                stage(combinedName) {
-                    build(version, label)
+        if (label.contains("docker")) {
+            def imageName = config["imageName"]
+            def combinedName = "${imageName}-${version}"
+            builders[combinedName] = {
+                node(label) {
+                    stage(combinedName) {
+                        docker.image(imageName).inside("-u root") {
```

Comment:
Because the docker image doesn't have the jenkins user. HOWEVER, that's only really required on the manylinux1 builds we do directly from the base container. Since we're generating our own here we should really update the dockerfile to add the user and then we don't need to do this nonsense.